fontforge/parsepdf.c

```diff
@@ -1893,7 +1893,9 @@ static SplineFont *pdf_loadtype3(struct pdfcontext *pc) {
 
     name=PSDictHasEntry(&pc->pdfdict,"Name");
     if ( name==NULL )
-	name=PSDictHasEntry(&pc->pdfdict,"BaseFont");
+        name=PSDictHasEntry(&pc->pdfdict,"BaseFont");
+    if ( name!=NULL )
+        name = copy(name+1);
```

Comment:
Can you add a `free` of name at the `fail` label

Reply:
Done